vassal-app/src/main/java/VASSAL/counters/PlaceMarker.java

```diff
@@ -651,13 +657,25 @@ protected Ed(PlaceMarker piece) {
         p.add(aboveLabel, aboveConfig);
         aboveConfig.getControls().setVisible(piece.matchRotation);
         aboveLabel.setVisible(piece.matchRotation);
+
+        copyConfig = createCopyConfig();
+        copyLabel = new JLabel(copyConfig.getName());
```

Comment:
Isn't `copyConfig.getName()` going to NPE because `createCopyConfig()` returns `null`?

Reply:
Technically yes, but we are in an `if (aboveConfig != null)` block and if createAboveConfig() has returned not null, then createCopyConfig() will also return not null.

Reply:
I don't follow. The only thing in the body of `createCopyConfig()` below is `return null;`.